CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased
+
+- New plugin config structure. New structure is an array of objects with `uri` and `config` keys. See [IPluginConfigMeta](./src/IPluginConfigMeta.ts).
```

Comment:
Mention that uri and config are required keys, and there are others optional keys like translation. When I first read it, I thought that there are only those two and none else

Reply:
Updated